src/assets/messages.json

```diff
@@ -1036,7 +1036,7 @@
     "originally": "Originally",
     "outgoing-calls": "Outgoing Calls",
     "output": "Output",
-    "output-branching": "Output Branching",
+    "exit-branching": "Exit Branching",
```

Comment:
Could you pls update our localization file ? https://docs.google.com/spreadsheets/d/136-y0DYRWFy6Max-Xz2AsqJ95TQvwrKjal6Wn-z0lsI/edit#gid=561815572

Reply:
Ah! I think I'm confused by what our process is for this right now :) 
I'm now understanding this as: We update manually, but in both places (gdoc/sheet + messages.json)?

Reply:
Done!